content/sensu-core/2.0/guides/clustering.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Clustering"
+linkTitle: "Clustering"
+weight: 120
+version: "2.0"
+product: "Sensu Core"
+platformContent: false
+menu: 
+  sensu-core-2.0:
+    parent: guides
+---
+
+## What is a Sensu Cluster?
+
+A Sensu Cluster is a group of [at least 3][1] sensu-backend nodes, each supporting an embedded or external etcd instance. Creating a Sensu Cluster ultimately configures an [etcd cluster][2].
+
+## Why use clustering?
+
+Clustering is important to make Sensu more highly available, reliable, and durable. It will help you cope with the loss of a backend node, prevent data loss, and distribute the network load of agents.
+
+## Configuring a cluster
+
+The sensu-backend arguments for its store mirror the [etcd configuration flags][3], however the Sensu flags are prefixed with `etcd`. For more detailed descriptions of the different arguments, you can refer to the [etcd docs][4] or sensu-backend help usage.
+
+You can configure a Sensu cluster in a couple different ways (we'll show you a few below) but it's recommended to adhere to some etcd cluster guidelines as well.
+
+> The recommended etcd cluster size is 3, 5 or 7, which is decided by the fault tolerance requirement. A 7-member cluster can provide enough fault tolerance in most cases. While larger cluster provides better fault tolerance the write performance reduces since data needs to be replicated to more machines. It is recommended to have an odd number of members in a cluster. Having an odd cluster size doesn't change the number needed for majority, but you gain a higher tolerance for failure by adding the extra member *(Core OS).*
```

Comment:
Even though this is a quote, let's fix the typo: "While larger cluster" --> "While a larger cluster"